### PR TITLE
mgr/dashboard: replace 'telemetry_notification_hidden' localStore to 1-year expiring cookie

### PR DESCRIPTION
**Pull request by Seremba**
When the user opts out of the Telemetry Module, it is recommendable to send a reminder after a certain period of time to 
     opt in. A cookie that expires after one year was implemented for the desirable results. 

      Fixes: https://tracker.ceph.com/issues/49580
      Signed-off-by: Seremba Patrick pserembae@gmail.com

## Checklist
- Tracker (select at least one)
  - [ ] References tracker ticket
  - [ ] Very recent bug; references commit where it was introduced
  - [ ] New feature (ticket optional)
  - [ ] Doc update (no ticket needed)
  - [ ] Code cleanup (no ticket needed)
- Component impact
  - [ ] Affects [Dashboard](https://tracker.ceph.com/projects/dashboard/issues/new), opened tracker ticket
  - [ ] Affects [Orchestrator](https://tracker.ceph.com/projects/orchestrator/issues/new), opened tracker ticket
  - [ ] No impact that needs to be tracked
- Documentation (select at least one)
  - [ ] Updates relevant documentation
  - [ ] No doc update is appropriate
- Tests (select at least one)
  - [ ] Includes [unit test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/tests-unit-tests/)
  - [ ] Includes [integration test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/testing_integration_tests/)
  - [ ] Includes bug reproducer
  - [ ] No tests

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`
- `jenkins test windows`
</details>

### CODE REVIEW

**Comment by ljflores** [comment]
Hey @Seremba, congrats on opening your first PR! A couple of initial details I see that should be addressed are 1) the commit titles and 2) signing off your commits. The Ceph project requires that you do these things in a specific way. Here are some guidelines here on how to do that: https://github.com/ceph/ceph/blob/master/SubmittingPatches.rst

Let me know if you have any questions! And look out for reviews on your implementation from the dashboard team.

**Comment by Seremba** [comment]
> Hey @Seremba, congrats on opening your first PR! A couple of initial details I see that should be addressed are 1) the commit titles and 2) signing off your commits. The Ceph project requires that you do these things in a specific way. Here are some guidelines here on how to do that: https://github.com/ceph/ceph/blob/master/SubmittingPatches.rst
> 
> Let me know if you have any questions! And look out for reviews on your implementation from the dashboard team.

Thank you so much! Let me try again. I appreciate your quick response.

**Comment by ljflores** [comment]
jenkins retest this please

**Comment by ljflores** [comment]
jenkins retest this please

**Comment by Seremba** [comment]
hi @ljflores, this is just an inquiry. After signing off all my commits, the test for signing off a commit still fails. Is there anything required on my side to pass that step? Thanks a lot.

**Comment by nizamial09** [comment]
@Seremba There are some unrelated commits I believe, in the commit section. Can you take care of that?

**Comment by Seremba** [comment]
Thanks a lot @nizamial09. Let me try again.
